### PR TITLE
Added Actions for building server

### PR DESCRIPTION
**Pull request by mytja**
This file now builds Go file & Docker file... You can see example run here: https://github.com/mytja-archive/ptrun/actions/runs/1139865571

I don't know how to integrate tests, so for now I won't integrate them...

I made a lot of commits 😅 , so I recommend you to squash them in a single commit...

### CODE REVIEW

**Comment by mytja** [comment]
Done. Everything is optimized...

File: .github/workflows/server.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          go-version: '^1.13.1' # The Go version to download (if necessary) and use.
+      - name: Get current Go version
+        run: go version
+      - name: Get Go dependencies
+        working-directory: ./server/
+        run: go mod download
+      - name: Build app
+        working-directory: ./server/
+        run: go build -o ptrun-server ./cmd/ptrun-server/main.go
+  docker:
```
Comment (davidmohar) [inline .github/workflows/server.yaml:29]: Can we make this run only on `develop` & `master` branches as a manual action so we don't push images to ghcr every time someone makes a PR and/or change there.

File: .github/workflows/server.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+jobs:
+  build:
+    name: Server build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1' # The Go version to download (if necessary) and use.
```
Comment (davidmohar) [inline .github/workflows/server.yaml]: We should be using `>=1.15`

**Review by davidmohar — CHANGES_REQUESTED** [review]
`server/tests/server_test.go` should be removed from this PR as it's not relevant to the overall story of this request. Otherwise everything else looks good and only needs minor improvements before we merge.

**Comment by mytja** [comment]
This has been fixed

**Comment by mytja** [comment]
https://github.com/mytja-archive/ptrun/actions/runs/1146131878
https://github.com/mytja-archive/ptrun/pkgs/container/ptrun%2Fserver

**Comment by mytja** [comment]
Everything is done
https://github.com/mytja-archive/ptrun/actions/runs/1146202183

**Review by davidmohar — APPROVED** [review]

**Pull request merged**